organisaatio-ui/src/components/Modaalit/YTJModaali/YTJBody.tsx

```diff
@@ -72,7 +72,7 @@ export default function YTJBody({ ytunnus, suljeModaali, setters }: Props) {
                         suffix={
                             input && (
                                 <Button variant={'text'} style={{ boxShadow: 'none' }} onClick={() => setInput('')}>
-                                    <Icon color={'#999999'} icon={clearIcon} />
+                                    <Icon fr={undefined} color={'#999999'} icon={clearIcon} />
```

Comment:
remove fr={}?

Reply:
this has creeped in so many places that it is truly annoying. 
reason is probably due to the updated packages but what the hell is it and why it needs to be explicitly set to undefined _everywhere_. If nothing else helps, maybe `<Icon>` should be wrapped somehow?
Lots of repetition due to this, not DRY

Reply:
i have no idea why ts says this is an error when fr key is not set on icon. This is some kind of temporary fix to make it compile and i cant find any info on how to fix this for now. I can wrap this to separate component as a temp fix?

Reply:
good starting step would be to find out why it's suddenly required and what it is for

Reply:
Sure. I googled it for quite the time and found almost nothing so i just tried to make ts compile. I can google some more if it would give some idea. Any help appreciated.

Reply:
Now there is a wrapper around and set 0 as it is the default value svg fr prop